src/lists/getList.test.ts

```diff
@@ -0,0 +1,181 @@
+import axios, { AxiosError, AxiosResponse } from "axios";
+import { getList } from "./getList";
+import { ListNotFoundError } from "../errors/ListNotFoundError";
+import {
+  EmailOctopusError,
+  ApiKeyInvalidError,
+  InvalidParametersError,
+  NotFoundError,
+  UnauthorisedError,
+} from "../emailOctopus";
+import { UnknownError } from "../errors/UnknownError";
+
+jest.mock("axios", () => {
+  const originalModule = jest.requireActual("axios");
+  return {
+    ...originalModule,
+    get: jest.fn(),
+  };
+});
+
+describe("getList", () => {
+  const FAKE_API_KEY = "1c2b40da-5fa3-11ee-8c99-0242ac120002";
+  const listId = "fakeListId";
+
+  describe("request successful", () => {
+    it("should return the list", async () => {
+      const expectedList = {
```

Comment:
For the mocked List could you use the type `List` [1]?

It would look like the following:

```tsx
import type { List } from "./types";

const expectedList: List = {
  id: "9e4ebe5b-a406-4c1a-af2c-fcd2c84e8e66",
  name: "Example list name",
  double_opt_in: false,
  fields: [],
  counts: {
    pending: 0,
    subscribed: 5,
    unsubscribed: 0
  },
  created_at: "2023-09-30T00:00:00+00:00"
};
```


[1] https://github.com/kartoffelkraft/email-octopus-ts/blob/main/src/lists/types.ts#L14C16-L14C16

Reply:
I adjusted the typing in the expected list